GNUmakefile

```diff
@@ -23,7 +26,11 @@ lint:
 
 .PHONY: fmt
 fmt:
-	gofmt -s -w $(GOFMT_FILES)
+	gofmt -s -w $(GOFMT_FILES); goimports -w $(GOFMT_FILES)
+
+.PHONY: addlicense
+addlicense:
+	addlicense -c "MongoDB Inc" $(GOFMT_FILES)
```

Comment:
super!

Reply:
we should probably call this in the precommit hook and apply the license automatically, rather than making it part of 'setup' which seldom gets called.

Reply:
It's there but with so many files changed easy to miss